movier.js

```diff
@@ -1,5 +1,5 @@
 const movier = require('movier');
 
-movier.searchTitleByName('interstellar').then(element => {
-    console.log(element);
+movier.searchTitleByName('cop out').then(element => {
```

Comment:
Is there a reason this is being hardcoded vs just basing in the movie title that the user is trying to add?